docs/paper/dev/api/event-api/event-listeners.mdx

```diff
@@ -42,7 +44,8 @@ public class ExampleListener implements Listener {
 :::note[Events]
 
 There is no list of events that can be listened to, however take a look 
-[here](https://jd.papermc.io/paper/1.20/org/bukkit/event/Event.html) to see all events that extend `Event`.
+<a href={"https://jd.papermc.io/paper/" + SoftwareVersionFetcher.getMajorPaperVersion() + "/org/bukkit/event/Event.html"}>here</a>
+to see all classes that extend `Event`.
```

Comment:
I think it would probably be very handy to have a component to make linking to specific javadocs easier. Just to avoid the boilerplate of `<a href=...`. It could take a parameter for the path to whatever, and the inner content would be the text to show instead like the a tag.